### ISSUE TITLE
Disable path conversion for Git Bash

### ISSUE THREAD

**Issue by Tantalus13A98B5F** [junegunn/fzf#1983]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [x] Windows
    - [ ] Etc.
- Shell
    - [x] bash (Git Bash/MSYS under mintty)
    - [ ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

It's the vim integration. 

1. Use the most recent Git for Windows (2.26.0)
2. Type `:FZF<CR>` in Vim
3. No new window of `cmd.exe` is started, and the window for vim itself would enter `cmd.exe`.

Seems the bash contained in Git for Windows is tranlating the `/C` arg for `cmd.exe` as a path (probably to `C:\`). The following patch would help (according to [this](https://github.com/git-for-windows/git/issues/577#issuecomment-166118846))

```diff
diff --git a/plugin/fzf.vim b/plugin/fzf.vim
index a84dcab..26dcaca 100644
--- a/plugin/fzf.vim
+++ b/plugin/fzf.vim
@@ -587,7 +587,7 @@ function! s:execute(dict, command, use_height, temps) abort
   elseif has('win32unix') && $TERM !=# 'cygwin'
     let shellscript = s:fzf_tempname()
     call writefile([command], shellscript)
-    let command = 'cmd.exe /C '.fzf#shellescape('set "TERM=" & start /WAIT sh -c '.shellscript)
+    let command = 'MSYS_NO_PATHCONV=1 cmd.exe /C '.fzf#shellescape('set "TERM=" & start /WAIT sh -c '.shellscript)
     let a:temps.shellscript = shellscript
   endif
   if a:use_height
```

**Comment by Konfekt** [comment]
While `MSYS_NO_PATHCONV=1` is one solution, why prefer it over doubling the first slash as alternatively proposed ?

**Comment by junegunn** [comment]
Fixed in #3798
